modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/http/HttpClient.java

```diff
@@ -142,6 +143,13 @@ public HttpResponse sendHttpRequest(HttpRequest request, String httpMethod) {
             }
 
             return new HttpResponse(responseCode, response.toString());
+        } catch (ConnectException e) {
+            BrokerClientException exception = new BrokerClientException();
+            exception.addMessage("Error calling broker https services.");
+            exception.addMessage("Please check whether the broker is running or the connectivity to the broker is not"
+                                 + " blocked through a firewall.");
```

Comment:
Shall we also specify the broker URL we are trying to connect?

Reply:
would a message like this be convenient 
`Please check whether the broker is running on (https://127.0.0.1:9000/broker/v1.0/) or the connectivity to the broker is not blocked through a firewall.`

Reply:
:+1: 
Do we need the parentheses around the URL? What about something like below. 

```
CLI tool cannot access https://127.0.0.1:9000/broker/v1.0/. Please check whether the broker is running or the connectivity to the broker is not blocked through a firewall.
```

@nilminiwso2